.github/workflows/gradle.yml

```diff
@@ -1,12 +1,13 @@
 name: Checks
-on: [push, pull_request]
+on: [ push, pull_request ]
 jobs:
   linux:
     name: PostgreSQL ${{ matrix.postgres }} (Linux)
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
Very useful, I didn't know it's possible to configure it in this way.

Reply:
It was really coincidence that I found that setting.